### PR TITLE
Added MIT-License

### LINKED ISSUE #3: Add license
Maybe MIT license? 🤔

### PR DESCRIPTION
**Pull request by TrungMa1**
Closes #3

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 ITHS Java23 https://github.com/Lucia560/Jakarta_labb.git
```
Comment (marcusju23) [inline LICENSE:3]: Remove .git

**Review by marcusju23 — CHANGES_REQUESTED** [review]
Nicely done! Minor change before approval.

**Review by Rishad11 — APPROVED** [review]

**Comment by TrungMa1** [comment]
Am i suppose to merge pull requests manually after approves?

**Comment by marcusju23** [comment]
> Am i suppose to merge pull requests manually after approves?

It depends on our team workflow. I think its good that someone else merges. So in my opinion, no self-mergning 😃

**Pull request merged**